build.sbt

```diff
@@ -31,12 +31,11 @@ lazy val root = (project in file("."))
     )
   .dependsOn(gatlingImap)
 
-// TODO enable WebsocketSimpleScenario and PushPlatformValidationScenario' websocketclose request when upgrade gatlingVersion
-val gatlingVersion = "3.9.5"
+val gatlingVersion = "3.11.3"
 
 scalacOptions := Seq("-unchecked", "-deprecation", "-feature", "-language:postfixOps", "-Ywarn-unused:imports",
       "-Wconf:msg=Auto-application to \\`\\(\\)\\` is deprecated:s")
 
 enablePlugins(GatlingPlugin)
 
-lazy val gatlingImap = ProjectRef(uri("https://github.com/linagora/gatling-imap.git"), "gatling-imap")
+lazy val gatlingImap = ProjectRef(uri("https://github.com/vttranlina/gatling-imap.git"), "gatling-imap")
```

Comment:
That way it should be marked as WIP.

Do not forget to update it back to `linagora` before merging.

Reply:
I merged the changes on gatling-imap, please rebase and remove this commit, and we should be good :)